build.rs

```diff
@@ -8,8 +8,19 @@ fn main() {
 
         println!("cargo:rustc-link-search=native={}", custom_lib_dir);
         println!("cargo:rustc-link-lib=static={}", custom_lib_name);
-        /* FIXME: this is assuming a C++ fuzzer, but should be customizable */
-        println!("cargo:rustc-link-lib=stdc++");
+
+        // Allow user to specify use of LLVM or GCC C++ stdlib, or none.
+        // "cargo:rustc-link-lib=stdc++" is used by default for
+        // backward-compatibility.
+        // Note: google/oss-fuzz uses LLVM C++ stdlib as of 15 Jan 2020
+        if let Ok(std_cxx_flag) = ::std::env::var("CUSTOM_LIBFUZZER_STD_CXX") {
```

Comment:
I think this is not going to do the intended default behavior if the env var is completely unset, and so `std::env::var` returns an `Err`.

Lets fix that by doing something like this:

```rust
match std::env::var("CUSTOM_LIBFUZZER_STD_CXX") {
    // Default behavior for backwards compat.
    Err(_) => println!("cargo:rustc-link-lib=stdc++"),
    Ok(s) if s == "none" => (),
    Ok(s) => println!("cargo:rustc-link-lib={}", s),
}
```